src/coreclr/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshal.CoreCLR.cs

```diff
@@ -780,12 +781,18 @@ public static object BindToMoniker(string monikerName)
             return obj;
         }
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2050:UnrecognizedReflectionPattern",
```

Comment:
These should be marked with `https://github.com/mono/linker/issues/1989`. We will want to revert these suppress messages once that issue is addressed.